src/lib.rs

```diff
@@ -31,6 +36,25 @@ mod blend;
 mod psd_channel;
 mod sections;
 
+/// An list of errors returned when processing PSD file.
+///
+/// This list is intended to grow over time and it is not recommended to exhaustively match against it.
```

Comment:
Can remove this and add `#[non_exhaustive]`

Reply:
Good point!